driver/src/main/scala/org/mongodb/scala/Subscription.scala

```diff
@@ -55,4 +55,11 @@ trait Subscription extends JSubscription {
    * @return `true` if this `Subscription` is currently unsubscribed, `false` otherwise
    */
   def isUnsubscribed: Boolean
+
+  /**
+   * Request the [[Observable]] to stop sending data and clean up resources.
+   *
+   * As this request is asynchronous data may still be sent to meet previously signalled demand after calling cancel.
+   */
+  override def cancel(): Unit = unsubscribe()
```

Comment:
The scala `Subscription` is very slightly different to the reactivestreams one - it has an `unsubscribe` method rather than a `cancel` method, it also has a `isUnsubscribed` method.